requirements-dev.txt

```
@@ -1,12 +1,17 @@
 # Internationalization and Localization requirements
--e git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6
-Django==1.11.25
+-e 'git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6 ; python_version == "2.7"'
+-e 'git://github.com/edx/xblock-sdk.git@0.2.2#egg=xblock-sdk==v0.2.2 ; python_version > "2.7"'
```

Newer versions can be installed straight from pip. 
```suggestion
xblock-sdk==0.2.2 ; python_version > "2.7"'
```

---

If installed from pip, we cannot install as editable. Makefile contains rules that expects xblock-sdk under ``.venv/src/xblock-sdk``

---

@shimulch, I checked the `Makefile` and found that one of the targets has commands to install the dependencies of the `xblock-sdk` from the editable directory. Since we are now installing it from PyPI, its dependencies should automatically get installed without issues, no?

---

@lgp171188, Yes for ``xblock-sdk==0.2.2``. But as we want to support ``python 2.7`` and ``django 1.11`` we need ``xblock-sdk==0.1.6`` which is not published on pypi. So we would've two different behavior which we might have to accommodate in Makefile (if py2 then install requirements from src else skip). Also we cannot install it as editable from pypi.

``ERROR: xblock-sdk is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).``